tests/functional/base_test.go

```diff
@@ -121,3 +121,37 @@ func GetSampleTopologySpec(label string) (map[string]interface{}, []corev1.Topol
 	}
 	return topologySpec, topologySpecObj
 }
+
+// GetExtraMounts - Utility function that simulates extraMounts pointing
+// to a  secret
+func GetExtraMounts(hemName string, hemPath string) []map[string]interface{} {
```

Comment:
if we use something like this in multiple operators, could add a test helper pkg to https://github.com/openstack-k8s-operators/lib-common/tree/main/modules/storage

Reply:
yeah, it's a good idea I agree